src/main/java/com/conveyal/gtfs/GTFSCache.java

```diff
@@ -248,6 +274,7 @@ private GTFSFeed retrieveFeed (String originalId) {
                 throw new RuntimeException(e);
             }
         } else {
+            LOG.warn("Feed {} found neither locally nor on S3", originalId);
```

Comment:
Well, this is reached when no file is found whatsoever, so I though it would be most descriptive to say `neither local nor s3`.